src/server/tests/routes.test.js

```diff
@@ -0,0 +1,10 @@
+const request = require('supertest');
+const app = require('../start');
+
+describe('Test Endpoints', () => {
+  it('Test root route', async() => {
+    const res = await request(app).get('/');
+    expect(res).toEqual('api working from digital ocean');
```

Comment:
Can you create a constant for the message string and reference it in the toEqual?? So if it changes in the app, we don't have to refactor the test

Reply:
Probably.  We could export it from the `routes` file and import it here.